flink-runtime/src/main/java/org/apache/flink/runtime/io/network/api/writer/SubtaskStateMapper.java

```diff
@@ -54,18 +54,6 @@
         }
     },
 
-    /**
-     * Discards extra state. Useful if all subtasks already contain the same information
-     * (broadcast).
-     */
-    DISCARD_EXTRA_STATE {
```

Comment:
I personally didn't find any problem with this removing but I just want to emphasize that if this enum is using somewhere in WebUI(or similar) it can not be so easily removed.

Reply:
Good point. It should not be a problem for this enum as it is not presented anywhere; it is an internal, strictly algorithmic class.